DEVX-2258 Check Prose for Offensive, Ableist and Gendered Language (#3405)
Nexmo/nexmo-developer: _documentation/en/messages/overview.md

The Messages API allows you to send and in some cases receive messages over SMS, MMS, Facebook Messenger, Viber, and WhatsApp. Further channels may be supported in the future.

> The following diagram illustrates how the Vonage Messages API enables you to send messages over multiple channels with minimal work:

Reviewer: @bencgreenberg I think I'd prefer something like "...via multiple channels from a single endpoint". That way we can state the benefit instead of just saying words to the effect of "it's easy". What do you think?

Author: I agree! If you'd like to modify it as a suggestion, I'll gladly commit it ❤️

Reviewer: @bencgreenberg Done! 😄

> The following diagram illustrates how the Vonage Messages API enables you to send messages over multiple channels with minimal work:

Reviewer: ```suggestion
The following diagram illustrates how the Vonage Messages API enables you to send messages via multiple channels from a single endpoint:
```
